src/web/components/SideBar.tsx

```diff
@@ -0,0 +1,156 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import {
+  makeStyles,
+  Button,
+  tokens,
+} from "@fluentui/react-components";
+import {
+  Home24Regular,
+  Home24Filled,
+  DataUsage24Regular,
+  DataUsage24Filled,
+  BookOpen24Regular,
+  BookOpen24Filled,
+  Code24Regular,
+  Code24Filled,
+  Database24Regular,
+  Database24Filled,
+  Cloud24Regular,
+  Cloud24Filled,
+  LearningApp24Regular,
+  LearningApp24Filled,
+  People24Regular,
+  People24Filled,
+  PanelLeftExpandRegular,
+  PanelLeftContractRegular,
+} from "@fluentui/react-icons";
+
+// Styles for the sidebar using Fluent UI's `makeStyles`
+const useStyles = makeStyles({
+    sidebar: {
```

Comment:
Can we follow a naming for the classNames? that way they dont collide with Fluent's. Something like ftk-componentName-sidebar